client/fresheyes-bot/src/index.ts

```diff
@@ -0,0 +1,58 @@
+import { Probot } from "probot";
+
+export = (robot: Probot) => {
+  robot.on(["pull_request.opened"], async (context) => {
+    /**  Get information about the pull request **/
+    const { owner, repo, pull_number: number } = context.pullRequest();
+    const pull_number = number ?? context.payload.number;
+
+    const { data } = await context.octokit.pulls.listReviewComments({ owner, repo, pull_number });
+
+    try {
+      if (!data || data.length === 0) {
+        return;
+      }
+
+      const groupComments: Record<string, Array<typeof data>> = data
+        .map((x: any) => ({ ...x, line: String(x.line) }))
+        .reduce((acc, curr) => {
+          const key = curr.line;
+
+          const group = acc[key] ?? [];
+
+          return { ...acc, [key]: [...group, curr] };
+        }, {});
+
+      await Promise.all(
+        Object.entries(groupComments).map(async ([k, v]) => {
```

Comment:
what's `k` and `v`. Lets use descriptive variable assignments

Reply:
I will make comments more descriptive